gcp/microservice-ecommerce/blueprint.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: xl/v1
+kind: Blueprint
+
+metadata:
+  projectName: GCP-Microservice-GKE-with-JHipster
+  description: |
+    The blueprint deploys an e-commerce microservice application to GCP GKE.
+    XL Deploy does the provisioning and deployment, while XL Release orchestrates everything.
+  author: XebiaLabs
+  version: 1.0
+
+spec:
+  parameters:
+  # General variables
+  - name: AppName
+    type: Input
+    description: What is the name of the application?
+
+  # GCP specific variables
+  - name: GCPProjectID
+    type: Input
+    description: What is the GCP project ID?
+  - name: K8SMasterUser
+    type: Input
+    description: What is the username for Kubernetes cluster?
+    default: admin
+  - name: K8SMasterPassword
+    type: Input
+    secret: true
+    description: What is the password for Kubernetes cluster?
```

Comment:
Is there a minimum length validation? I got the following error:

* module.gke.google_container_cluster.primary: 1 error(s) occurred:
* google_container_cluster.primary: googleapi: Error 400: Cluster.master_auth.password must be at least length 16., badRequest